pages/papers/how-to-practices-and-learn-ruby-tests.mdx

```diff
@@ -4,38 +4,34 @@ lang: en
 description: From beginner to expert in ruby tests.
 ---
 
-<div css={{ color: "red", fontStyle: "italic", fontSize: "2em" }}>
-  In Progress
-</div>
-
 **Tags:** _Ruby_, _Tests_
 
-**Audience Level:** _~~Beginer~~_ _~~All~~_ _Intermediate_
+**Audience Level:** _Beginer_
 
 ### Elevator Pitch (300 characters)
 
 <div css={{ color: "#888", fontStyle: "italic", fontSize: "0.8em" }}>
 You have 300 characters to sell your talk. This is known as the "elevator pitch". Make it as exciting and enticing as possible.
 </div>
 
-Cuando me inicie en el mundo de la programación los tests fueron un gran punto de partida para reducir mi frustración en el proceso de aprendizaje y coger confianza sobre el código que escribía, ahora en ruby, perfeccione esas habilidades, y te mostrare, como aprender ruby y aprender test! al mismo tiempo!
+When I started in the world of programming the tests were a great starting point to reduce my frustration in the learning process and gain confidence in the code I wrote, now in ruby, perfect those skills, and I will show you how to learn ruby and learn test! at the same time!
```

Comment:
the world of programming, the tests ** [...]. Now in Ruby, I perfected those skills, and I will show you how to learn Ruby and learn tests! At the same time!